exercises/concept/luigis-luscious-lasagna/.meta/Exemplar.elm

```diff
@@ -0,0 +1,13 @@
+module LuigisLusciousLasagna exposing (remainingTimeInMinutes)
+
+
+remainingTimeInMinutes : Int -> Int -> Int
+remainingTimeInMinutes layers minutesSinceStarting =
+    let
+        expectedMinutesInOven =
+            40
+
+        preparationTimeInMinutes =
+            2 * layers
+    in
+    preparationTimeInMinutes + expectedMinutesInOven - minutesSinceStarting
```

Comment:
There's not much too this exercise, but that is fine in this case I think.
One thing to consider is how to verify that the student is actually using a `let` expression, and not several functions. You'll probably need to build an analyzer for that.